site/docs/v1/tech/themes/tailwind.adoc

```diff
@@ -0,0 +1,208 @@
+---
+layout: doc
+title: Tailwind CSS
+description: Detailed documentation for integrating Tailwind CSS in FusionAuth.
+navcategory: customization
+---
+:page-liquid:
+
+== Overview
+
+Tailwind CSS is a utility-first CSS framework for rapidly building custom user interfaces. If you are already using Tailwind CSS in your application, you can integrate it into FusionAuth.
+
+== Prerequisites
+
+* Node.js 18 or later
+
+== Setting up FusionAuth
+
+Create a new theme by navigating to [breadcrumb]#Customization -> Themes# and then click the [uielement]#Add# button. Enter a [field]#Name# for the theme and click the [uielement]#Save# button.
+
+image::themes/tailwind/create-theme.png[Create a theme,width=1200]
+
+Create a new API key with permissions to modify the theme. You can do this by going to [breadcrumb]#Settings -> API Keys#. Then click the [uielement]#Add# button and select `GET` and `PATCH` permissions for `/api/theme/`.
+
+image::themes/tailwind/create-api-key.png[Create an API key,width=1200]
+
+== Installation
+
+We are going to use the FusionAuth Theme Helper project as a starting point.
+
+First, clone the FusionAuth Theme Helper project:
+```bash
+git clone https://github.com/FusionAuth/fusionauth-theme-helper
+```
+
+Copy the `.env.example` file to `.env` and update the values:
+```bash
+API_KEY=OjgmZzp5Cg_osynwHOATFV9368lRlP0e_imhsTtl5arxym9ymhHvKQ_o # the API key with permissions to modify the theme
+THEME_ID=5d175a65-71f2-489a-a6d2-89adad4f5b00 # your theme's UUID
+FUSIONAUTH_URL=http://localhost:9011 # your FusionAuth server, including the protocol and hostname, but no paths
+TMP_DIR=src # a temporary directory; this will be created
+```
+
+To initially download the theme from FusionAuth, run the following command:
+```bash
+./download.sh
+```
+
+Next, install the dependencies and add tailwindcss:
+```bash
+npm install
+npm install tailwindcss@latest
```

Comment:
Do they promise backward compatibility? Will `latest` always be safe to use?

Reply:
I pinned the version to `@3`